src/main.js

```diff
@@ -0,0 +1,16 @@
+import InfoView from './view/info-view';
+import { render, RenderPosition } from './render.js';
+import { EventListPresenter } from './presenter/index.js';
+import { FilterView, SortView} from './view/index.js';
```

Comment:
index.js не обязательно писать, js автоматом подхватит этот файл. 

```suggestion
import { EventListPresenter } from './presenter';
import { FilterView, SortView} from './view';
```